Bottom: aa146da8000e907c609817c3c72e9dd3c4ee22de
Top:    45f603b1ec7c7f13219406194d1d5b87cd00d3c0
Author: Peter Clifton <peter@clifton-electronics.co.uk>
Date:   2016-02-28 03:58:13 +0000

Disable debug hack


---

diff --git a/src/hid/step/step.c b/src/hid/step/step.c
index 0a325f8..52182c0 100644
--- a/src/hid/step/step.c
+++ b/src/hid/step/step.c
@@ -152,7 +152,7 @@ step_do_export (HID_Attr_Val * options)
   if (filename == NULL)
     filename = "pcb-out.step";
 
-#if 0
+#if 1
   board_outline_list = object3d_from_board_outline ();
 #else
   board_outline_list = NULL;
